server/members/migrations/0004_user_members_use_usernam_82d39a_idx_and_more.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 4.2.16 on 2024-11-30 23:07
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('members', '0003_user_profile_picture_url'),
+    ]
+
+    operations = [
+        migrations.AddIndex(
+            model_name='user',
+            index=models.Index(fields=['username'], name='members_use_usernam_82d39a_idx'),
```

Comment:
ahh wait i recall you creating an index in prod at some point using raw SQL. If so, I think the migration will fail so I'll keep an eye out while applying, and if so I'll try to remove index from prod so all indices are defined with the model

otherwise was it declared somewhere in the code? might be missing it

Reply:
It should already be in a migration, I didn't do it raw.

Reply:
[It's here](https://github.com/swecc-uw/swecc-server/blob/main/server/members/migrations/0002_add_lowercase_indexes.py)

Reply:
ah i see, how did you add it? i don't see it defined in the model anywhere

Reply:
I just wrote it, I did'nt know you can define it in the model lol